python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -517,3 +561,52 @@ def _private_display(self, reg_slice, this_format, default_width, markup):
             ds += markup(this_format % tuple(vals), colour)
 
         return ds
+
+    @staticmethod
+    def jobHasChildrenTest(jobDirectory, datafileName):
```

Comment:
Given it's running the test for the children existing and this triggers the test to run I'd prefer to have this name, I'll update the docs to reflect this